src/App.jsx

```diff
@@ -60,9 +60,20 @@ export function App() {
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [listToken]);
 
+	function logOut() {
+		const confirm = window.confirm(
+			`If you're ready to log out, make sure to write down your list name before you click ok! It is ${listToken}`,
+		);
+		if (confirm) {
+			localStorage.clear();
```

Comment:
I like the progress - we learned about the window.confirm in one of the previous tickets and now you know the structure and are already applying it in a different place